lib/reporters/cli/index.js

```diff
@@ -151,15 +151,18 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
 
     // output the response code, reason and time
     emitter.on('request', function (err, o) {
-        if (err) { return; }
+        if (err) {
+            print.lf(colors.red('[errored]'));
```

Comment:
Could you also add the error message after `[errored]`? That will make debugging a whole lot easier 😄

Reply:
I was trying to keep it consistent with what was there before, but that improvement makes sense. I'll send an update shortly.